sensors/Makefile

```diff
@@ -22,7 +22,8 @@ SENSORS_SIM := imu/imu_sim.c mag/mag_sim.c baro/baro_sim.c gps/gps_sim.c gps/gps
 SENSORS_LOCAL := $(SENSORS_SIM) $(addsuffix .c, $(filter $(SENSORS_ALL), $(DEVICE_SENSORS)))
 
 NAME := sensors
-LOCAL_HEADERS_DIR := nothing
+
+LIBS := $(SENSORS_PROJECT_DEPS) # Project-specific sensors .a files and their dependencies
```

Comment:
not related to this change, but:
 - `SENSORS_SIM` files are included twice in `LOCAL_SRCS` .
 - IMHO we should use wildcards here instead of hand-listing every c file

Reply:
- `SENSORS_SIM` fix included in one of commits as it is a quick fix
 - added commits that use wildcards for sensor selection